pydantic/typing.py

```diff
@@ -48,11 +48,13 @@ def evaluate_forwardref(type_, globalns, localns):  # type: ignore
 
     AnyCallable = TypingCallable[..., Any]
 
-try:
-    from typing_extensions import Literal
-except ImportError:
-    Literal = None  # type: ignore
-
+if sys.version_info < (3, 8):
```

Comment:
Needed to use the `sys.version_info` check for mypy compatibility with both 3.8 and <=3.7